### PR TITLE
Add function support

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: exec/csnex/Exec.cs [hunk 1]
```diff
@@ -11,27 +11,32 @@ public Executor(Bytecode bc)
...
+            Callstack = new Stack<int>();
             global = new Global(this);
             bytecode = bc;
+            ParamRecursionLimit = 2000; // ToDo: Add runtime$setRecursionLimit()
         }
 
         private int exit_code;
         private readonly Bytecode bytecode;
         public Stack<Cell> stack;
+        private Stack<int> Callstack;
```
Comment (ghewgill) [inline exec/csnex/Exec.cs:23]: The capitalization style here seems inconsistent. Should this be `callstack`?

File: exec/csnex/Exec.cs [hunk 2]
```diff
@@ -11,27 +11,32 @@ public Executor(Bytecode bc)
...
         public bool enable_assert;
         private int ip;
         private Global global;
+        private Int32 ParamRecursionLimit;
```
Comment (ghewgill) [inline exec/csnex/Exec.cs:27]: Also inconsistent capitalization style here.

**Review by ghewgill — CHANGES_REQUESTED** [review]

File: exec/csnex/Exec.cs (hunk 1 above)
Comment (gitlarryf) [reply exec/csnex/Exec.cs]: Fixed.

File: exec/csnex/Exec.cs (hunk 2 above)
Comment (gitlarryf) [reply exec/csnex/Exec.cs]: Fixed.

**Pull request merged**